consoles/vnc_base.pm

```diff
@@ -23,11 +23,15 @@ use feature 'say';
 use base 'consoles::network_console';
 
 use consoles::VNC;
-use Time::HiRes qw(usleep gettimeofday);
+use List::Util 'max';
+use Time::HiRes qw(usleep time);
 
 use Try::Tiny;
 use testapi 'get_var';
 
+# speed limit: 30 keys per second
+use constant VNC_TYPING_LIMIT_DEFAULT => 30;
```

Comment:
Would you mind putting your PR description comment rather in here to explain the value selection? And are you sure we want to loose the speed benefit when "mistyping" now seems to happen only if either modifier keys are involved or it's a clearly special product, e.g. the bootloader line.

Reply:
I'm not so sure about this anymore. 30 keys a second is still very fast - and we hopefully see less mistypings on e.g. gnome terminals the more realistic our input is.

And I don't believe in code comments to explain numbers - they outdate super fast and the git history is so much more reliable.